vividus-plugin-mobile-app/src/main/java/org/vividus/mobileapp/action/MobileSequenceActionType.java

```diff
@@ -0,0 +1,177 @@
+/*
+ * Copyright 2019-2022 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.vividus.mobileapp.action;
+
+import java.lang.reflect.Type;
+import java.time.Duration;
+
+import org.openqa.selenium.Point;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.interactions.PointerInput;
+import org.vividus.steps.ui.model.SequenceActionType;
+
+public enum MobileSequenceActionType implements SequenceActionType
+{
+    PRESS(true)
+    {
+        @Override
+        public void addAction(Actions actions, Object argument)
+        {
+            performOnWebElement(argument, (WebElement e) -> {
+                setActivePointer(actions);
+                actions.clickAndHold(e);
+            }, () -> {
+                setActivePointer(actions);
+                actions.clickAndHold();
+            });
+        }
+
+        @Override
+        public Type getArgumentType()
+        {
+            return WebElement.class;
+        }
+    },
+    TAP(true)
+    {
+        @Override
+        public void addAction(Actions actions, Object argument)
+        {
+            performOnWebElement(argument, (WebElement e) -> {
+                setActivePointer(actions);
+                actions.click(e);
```

Comment:
I don't think we should use web actions based on mouse.
See:
- https://appiumpro.com/editions/29-automating-complex-gestures-with-the-w3c-actions-api
- https://appiumpro.com/editions/30-ios-specific-touch-action-methods
- https://github.com/appium/appium-uiautomator2-driver/blob/master/docs/android-mobile-gestures.md
- https://github.com/appium/appium-uiautomator2-driver/blob/master/docs/actions.md
- https://github.com/appium/appium-xcuitest-driver#platform-specific-extensions